apps/berik-inntektsmelding-service/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/berikinntektsmeldingservice/BerikInntektsmeldingService.kt

```diff
@@ -98,7 +103,7 @@ class BerikInntektsmeldingService(
             erDuplikat = Key.ER_DUPLIKAT_IM.les(Boolean.serializer(), melding),
         )
 
-    override fun utfoerSteg0(
+    override fun utfoerSteg0( // TODO: Forespørsel sendes inn, trenger ikke slå opp!?
```

Comment:
Det stemmer. Det var en relativt ny endring (for oss som har hatt ferie) å hente i forrige service. Nice catch!

Reply:
Jeg fikser dette for vanlig innsending nå.